### DISCUSSION TITLE
Add password manager for restricted sites

### DISCUSSION THREAD

**Discussion by rayman3003** [setvisible/DownZemAll#216] [Ideas]
Could u please add a password-manager inside the DownZemAll's options, so we could add & save username & password for the links that requires username/pass?

**Comment by setvisible** [comment]
Hi,
If what you mean is a proxy username/password, it's already there, in Preference > Network > Proxy
Regards

**Comment by rayman3003** [comment]
> If what you mean is a proxy username/password, it's already there, in Preference > Network > Proxy
> Regards

No, I didn't mean that.

Here is an example from IDM:

https://www.internetdownloadmanager.com/register/new_faq/sites2_5.html

**Comment by setvisible** [comment]
Looks similar to #25
